test/cpp/test_aten_xla_tensor_1.cpp

```diff
@@ -1470,6 +1470,34 @@ TEST_F(AtenXlaTensorTest, TestStack) {
   }
 }
 
+TEST_F(AtenXlaTensorTest, TestHstack) {
```

Comment:
Can we also check the counter in the test, so we can make sure the op is lowered instead falling back to cpu

Reply:
Good point. I was thinking the XLA op will automatically throws out the error if 1) the op lowering is not implemented and also 2) the input tensor is copied to the device. Looks like this is not the case. Thanks